fe/fe-core/src/main/java/com/starrocks/catalog/HiveTable.java

```diff
@@ -414,47 +413,54 @@ private void validate(Map<String, String> properties) throws DdlException {
         }
     }
 
-    private Set<PrimitiveType> getValidColumnType(String hiveType) {
+    private boolean validateColumnType(String hiveType, Type type) {
```

Comment:
It seems that it's better to create HiveType class.

Reply:
Now we just do some type match validation, but I agree to create HiveType class if we have further use case on it.